doc/source/rest.j2

```diff
@@ -108,9 +149,11 @@ to be handled:
    Depending on the amount of data that is unprocessed, `refresh` may add
    some overhead to your query.
 
-The list of points returned is composed of tuples with (timestamp,
-|granularity|, value) sorted by timestamp. The |granularity| is the |timespan|
-covered by aggregation for this point.
+Filter
```

Comment:
I consider `Filter` should be the subsection of `Read` since it is used when reading aggregates